### ISSUE TITLE
Support Ubuntu 22.04 on AT next

### ISSUE THREAD

**Issue by er-1** [advancetoolchain/advance-toolchain#2704]

**Comment by er-1** [comment]
I'm facing an issue with `mpfr`:
```
$ cat logs/_mpfr_1-3_standard_buildf-02_pre_configure.log
+ set -x
+ eval atcfg_pre_configure
++ atcfg_pre_configure
++ cc_64=/usr/bin/gcc
++ cxx_64=/usr/bin/g++
++ [[ no != \y\e\s ]]
++ prefix=opt/at-next-16.0-0-alpha
++ libdir=/opt/at-next-16.0-0-alpha/lib
++ [[ ppc64le == ppc64* ]]
++ [[ 64 == \3\2 ]]
++ libdir=/opt/at-next-16.0-0-alpha/lib64
++ [[ ! -e build/at16.0-0-alpha.ubuntu-22_ppc64le_ppc64le/sources/mpfr/configure ]]
++ pushd build/at16.0-0-alpha.ubuntu-22_ppc64le_ppc64le/sources/mpfr
++ ./autogen.sh
autoreconf: export WARNINGS=all,error
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: aclocal --force -I m4
autoreconf: running: /usr/bin/autoconf --force
configure.ac:550: warning: The macro `AC_HEADER_TIME' is obsolete.
configure.ac:550: You should run autoupdate.
./lib/autoconf/headers.m4:743: AC_HEADER_TIME is expanded from...
acinclude.m4:38: MPFR_CONFIGS is expanded from...
configure.ac:550: the top level
autoreconf: error: /usr/bin/autoconf failed with exit status: 1
./autogen.sh: restoring the INSTALL file
./autogen.sh: restoring the doc/texinfo.tex file
++ return 1
+ ret=1
+ set +x
```

`autogen.sh` is [here](https://gitlab.inria.fr/mpfr/mpfr/-/blob/4.1.0/autogen.sh)
`autoconf` version is 2.71.

**Comment by er-1** [comment]
This is fixed by #2730
